image-index.md

```diff
@@ -67,7 +67,12 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`features`** *array of strings*
 
-        This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
+        This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature.
+
+        When `architecture` is 386 or amd64 Image indexes SHOULD use, and implementations SHOULD understand, values [supported by Linux][cpufeatures.h] with the `X86_FEATURE_` prefix removed and the remainder lowercased (e.g. `fpu` for `X86_FEATURE_FPU` and `vme` for `X86_FEATURE_VME`).
```

Comment:
> These should not be defined in terms of linux, but instead in terms of amd64/x86.

Isn't that what I'm doing?  As I said in the [initial post][1], I'm happy to link to something other than a Linux header if we find an OS-agnostic registry, but I'm currently not aware of one.



[1]: https://github.com/opencontainers/image-spec/pull/631#issue-218396244